Libraries/Components/Touchable/__tests__/__snapshots__/TouchableWithoutFeedback-test.js.snap

```diff
@@ -15,3 +15,94 @@ exports[`TouchableWithoutFeedback renders correctly 1`] = `
   Touchable
 </Text>
 `;
+
+exports[`TouchableWithoutFeedback with disabled state should be disabled when disabled is true 1`] = `
+<View
+  accessibilityState={
+    Object {
+      "disabled": true,
+    }
+  }
+  accessible={true}
+  focusable={false}
+  onClick={[Function]}
+  onResponderGrant={[Function]}
+  onResponderMove={[Function]}
+  onResponderRelease={[Function]}
+  onResponderTerminate={[Function]}
+  onResponderTerminationRequest={[Function]}
+  onStartShouldSetResponder={[Function]}
+/>
+`;
+
+exports[`TouchableWithoutFeedback with disabled state should be disabled when disabled is true and accessibilityState is empty 1`] = `
+<View
+  accessibilityState={Object {}}
```

Comment:
It looks like there's a couple of cases where accessibilityState is not including `disabled: true` when it should (including this one) - perhaps you need to re-run the snapshot tests or investigate in the code why this is the case?

Reply:
Thanks for the review, I'm going to take a look at it!

Reply:
Found the issue, it's related with this code of TouchableWithoutFeedback:

https://github.com/facebook/react-native/blob/08ef1df1bad8408a75a195dda698db184efea0b7/Libraries/Components/Touchable/TouchableWithoutFeedback.js#L122-L126